src/test/scala/mesosphere/marathon/core/deployment/impl/TaskStartActorTest.scala

```diff
@@ -212,6 +216,31 @@ class TaskStartActorTest extends AkkaUnitTest with Eventually {
       expectTerminated(ref)
     }
   }
+
+  "relaunch when resident task gets terminal" in {
```

Comment:
this test was failing prior to the fix...

Reply:
When did this test fail last time?

Reply:
I think what Alena means is that this new test fails if the fix in this PR is not applied to the code.

Reply:
Exactly :)